virttest/tests/unattended_install.py

```diff
@@ -1142,6 +1144,35 @@ def string_in_serial_log(serial_log_file_path, string):
         return False
 
 
+def attempt_to_log_useful_files(test, vm):
+    """
+    Uses "vm.serial_console" to look in the system for useful log files
+    """
+    console = vm.serial_console
+    for path_glob in ["/*.log", "/tmp/*.log", "/var/tmp/*.log"]:
+        try:
+            paths = console.cmd("ls -1 %s" % path_glob)
+        except (aexpect.ExpectError, aexpect.ShellError):
```

Comment:
We should also handle other type of errors in this function, otherwise, we will miss the culprit which directly failed the test, so the code would be like:

```python
for path_glob in ["/*.log", "/tmp/*.log", "/var/tmp/*.log"]:
    try:
        try:
            paths = console.cmd("ls -1 %s" % path_glob)
        ...
    except Exception as details:
        logging.error("Failed to get '%s' from guest: %s", path_glob, str(details))
```

Reply:
OK, that seems like a reasonable request. Let me modify it a bit and update...